src/interface/playerSelection/PlayerTile.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+
+
+import RoleIcon from 'common/RoleIcon';
+
+
+class PlayerTile extends React.PureComponent {
+  static propTypes = {
+    player: PropTypes.object.isRequired,
+    selectedPlayer: PropTypes.object,
+    analysisUrl: PropTypes.string.isRequired,
+    handleClick: PropTypes.func.isRequired,
+    history: PropTypes.object.isRequired,
+  };
+
+  timeout = null;
+
+  render() {
+    const { player, selectedPlayer, analysisUrl, handleClick, history } = this.props;
+    return (
+      <span
+        key={player.guid}
+        onClick={() => { this.timeout = setTimeout(function(){ handleClick(); }, 200); }}
+        onDoubleClick={() => { clearTimeout(this.timeout); history.push(analysisUrl); }}
+        className={`player${selectedPlayer && selectedPlayer.guid === player.guid ? ' selected' : ''}`}
```

Comment:
Please use curly brackets around ternaries with multiple conditions as operator priority in JS is weird and omitting curly brackets might lead to bugs.